tests/unit/datarobot_drum/drum/test_main.py

```diff
@@ -23,7 +23,6 @@ def test_custom_model_workers(
     options = argparse.Namespace()
     options.max_workers = 0
```

Comment:
this should be the string "0", correct?  if you're reading from env vars, then it won't be cast to an int.

Reply:
Runtime params are type aware and support numeric types and casts them (to floats actually I think).